### PR TITLE
Update thumbnail-util guide

### PR DESCRIPTION
**Pull request by clockr63**
1. Remove deprecated async API
2. add new sync API

### CODE REVIEW

File: docs/application/native/guides/multimedia/thumbnail-images.md [hunk 1]
```diff
@@ -92,65 +97,27 @@ To enable your application to use the thumbnail util functionality:
...
-2. Set the path to the file using the `thumbnail_util_set_path()` function:
-
-   ```
-   ret = thumbnail_util_set_path(g_thumb_h, image_test_path);
-   ```
-
-   The function binds the thumbnail utility handle (first parameter) with the file specified in the path variable (second parameter).
-
-3. Set the thumbnail size using the `thumbnail_util_set_size()` function:
+1.1. To receive the results in a file, use the `thumbnail_util_extract_to_file()` function:
```
Comment (journey2w) [inline docs/application/native/guides/multimedia/thumbnail-images.md]: 1.1. is not list tag of markdown.

Please change it to 1. , 2. , 3., ...

**Comment by clockr63** [comment]
Update as your comment.
Please review again.

**Comment by mijin85cho** [comment]
Hello,  nagatejar. 
If you see no other issues, please approve this PR. When this PR is approved, I'll merge it.

Regards

**Comment by mijin85cho** [comment]
Hello, @nagatejar.

Would you approve this PR if everything is ok to go?

Regards

File: docs/application/native/guides/multimedia/thumbnail-images.md [hunk 2]
```diff
@@ -81,76 +83,41 @@ To enable your application to use the thumbnail util functionality:
...
-   ```
-   thumbnail_util_destroy(g_thumb_h);
-   ```
+   You get a BGRA color image.
 
 ## Related Information
 - Dependencies
-  - Tizen 2.4 and Higher for Mobile
-  - Tizen 3.0 and Higher for Wearable
+  - Tizen 5.0 and Higher
```
Comment (mijin85cho) [inline docs/application/native/guides/multimedia/thumbnail-images.md]: Hello, clockr63 

Would you distinguish the version information for Mobile and Wearable?
Like below

```
  - Tizen 5.0 and Higher for Mobile
  - Tizen 5.0 and Higher for Wearable
```

Also I think `Tizen 5.0 and Higher` can make an user confused 
as it sounds like Tizen 3.0 and 4.0 don't support thumbnail.

How about leaving it as `Tizen 3.0`, 
and add a note to updated part for Tizen 5.0, like "this functions is supported from 5.0"?

Thanks in advance.

Comment (clockr63) [reply docs/application/native/guides/multimedia/thumbnail-images.md]: All existing functions have been deprecated. 
This guide has only new functions(for 5.0).

And these functions support all profiles. 
So, there is no need to distinguish the profile.

Comment (mijin85cho) [reply docs/application/native/guides/multimedia/thumbnail-images.md]: For the version information, in that case, 5.0 would be OK.

For the profile information,
would you consider the consistency with other guide pages?
That format is to show the profile information for whole guide pages,
and for now this thumbnail-images.md page only is using different form.

**Comment by clockr63** [comment]
@mijin85cho Update the guide as your comment.

**Comment by annie-samsung** [comment]
Hello Divya,
Could you verify if all of Nagateja's comments are fixed and if the file looks OK?

**Comment by mijin85cho** [comment]
Thanks for the update & review

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**